packages/admin-ui-docs/docs/guidelines/components/search.mdx

```diff
@@ -85,11 +184,13 @@ function Example() {
 }
 ```
 
-### Persisted state in query params
-
-You can persist the search value within the browser's query params using the `useQuerySearchState` hook, instead of `useSearchState`. The application should be wrapped by `QueryStateProvider`.
+</PageSection>
 
-_watch how your url changes (Note that the persisted value is the debounce value)_
+<PageSection
+  id={tabToc.overview[5].id}
+  title={tabToc.overview[5].value}
+  explanation="You can persist the search value within the browser's query params using the `useQuerySearchState` hook, instead of `useSearchState`. The application should be wrapped by `QueryStateProvider`. Watch how your url changes (Note that the persisted value is the debounce value)."
```

Comment:
```suggestion
  explanation="You can persist the search value within the browser's query params using the `useQuerySearchState` hook, instead of `useSearchState`. The application should be wrapped by `QueryStateProvider`. Watch how your url changes (note that the persisted value is the debounced value)."
```